docs/reference/commandline/cli.md

```diff
@@ -171,7 +185,12 @@ Following is a sample `config.json` file:
       "imagesFormat": "table {{.ID}}\\t{{.Repository}}\\t{{.Tag}}\\t{{.CreatedAt}}",
       "statsFormat": "table {{.Container}}\t{{.CPUPerc}}\t{{.MemUsage}}",
       "serviceInspectFormat": "pretty",
-      "detachKeys": "ctrl-e,e"
+      "detachKeys": "ctrl-e,e",
+      "credsStore": "secretservice",
+      "credHelpers": {
+        "kontainr.ly": "hip-star",
+        "sfo.unicorn.com": "vcbait"
```

Comment:
for documentation, perhaps use the `example.*` domain's, which are intended for that (the domains here may actually exist at some point). e.g., `my-registry.example.com`